Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -29,6 +29,26 @@ Import-Module "$PSScriptRoot\DeploymentUtilities\Microsoft.TeamFoundation.Distri
 # Telemetry
```

Comment:
ensure you hold this PR till you update the DeploymentUtilities.zip , else the empty values will get publish as part of Publish-Azure-Telemetry

Reply:
Updated DeploymentUtilities.zip url and patch version